### PR TITLE
Set up tachyons, remove bootstrap, add custom colors

### PR DESCRIPTION
**Pull request by Cleop**
#173 Removes bootstrap, adds tachyons
#182 Adds dwyl-site colors
#170 Adds favicon

### CODE REVIEW

File: web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -6,28 +6,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="">
-
-    <title>Hello Tudo!</title>
+    <title>Help Wanted - dwyl</title>
```
Comment (Jbarget) [inline web/templates/layout/app.html.eex:9]: I thought @harrygfox mentioned avoiding having "dwyl" in the heading of the page

**Review by samhstn — APPROVED** [review]
👍

**Review by Jbarget — CHANGES_REQUESTED** [review]

File: web/templates/layout/app.html.eex (hunk 1 above)
Comment (Cleop) [reply web/templates/layout/app.html.eex:9]: I think that was for the page itself rather than the tab...

Comment (Jbarget) [reply web/templates/layout/app.html.eex:9]: ignore me :)

**Review by Jbarget — APPROVED** [review]

**Pull request merged**